package.json

```diff
@@ -22,6 +22,12 @@
   "license": "MIT",
   "readmeFilename": "README.md",
   "gitHead": "dcde154040d17d321664a7efc38e8531330335f9",
+  "tap": {
```

Comment:
I don't understand what this does and why it needs to be in the package.json .. is it maybe tap adding it without us doing this explicitly?

Reply:
I added it explicitly - it doesn't necessarily need to be added here, if you think a `.taprc` or another option is better, happy to do that: https://node-tap.org/docs/configuring/

Reply:
Well it wasn't needed before why would it be now? Is there anything in the config needed to make this work in CI?
Haven't used tap in a while (using tape instead) so might be out of touch with what's used here.

Reply:
Because with the CI re-enabled, the `tap` check was failing with a non-zero exit code due to missing coverage. Here you can see that in my fork: https://github.com/kevin-david/anchor-markdown-header/runs/7707974419?check_suite_focus=true#step:5:140

Reply:
Hey sorry for the delay. I'd feel better with a `.taprc` instead.
So please just adapt it and we can merge.

Reply:
Done with https://github.com/thlorenz/anchor-markdown-header/pull/46/commits/3ad770caeec8a6ac661428319705a3a10861c184